### DISCUSSION TITLE
Removing images from the index

### DISCUSSION THREAD

**Discussion by 92amartins** [policratus/pupyl#70] [Ideas]
@policratus, does `pupyl` offer the ability to remove all the images in an index (flush)? What if I wanted to remove a specific image from the index, is that possible?

While testing `pupyl`, I find that these capabilities would be useful if one decides not to index a particular image for example.

**Comment by policratus** [comment]
Another great idea @92amartins! There's support for removing images, but it's limited: you can remove from the index, but not from the database. Better explaining, the tensor can be removed from a index, but all its references (image copy, metadata, etc) will be still on the data directory.

I'm creating a card for this feature, definitely will be part of release 0.10.7

**Comment by policratus** [comment]
Hey @92amartins, how's it going? Just merged the pull request #111 that implements this idea! Will be part of `pupyl` version `0.12.4`.

**Reply by 92amartins** [reply]
Awesome :tada: :tada:
